common/archive/archive.go

```diff
@@ -25,6 +25,7 @@ type CollectionMetadata struct {
 	Collection string `bson:"collection"`
 	Metadata   string `bson:"metadata"`
 	Size       int    `bson:"size"`
+	Type       string `bson:"type"`
```

Comment:
Is the docstring's assertion that `There is one CollectionMetadata per collection that will be in the archive` still true? Or will these be handled like the intents, where there is one `CollectionMetadata` that covers both?

Reply:
Depends how you interpret "collection that will be in the archive". There is still one `CollectionMetadata` per collection that will appear in the data blocks. For a timeseries collection T, there will be no data blocks for T, instead there will be system.buckets.T. So there is a one-to-one mapping in that sense (although the namespace names are different).

However if you're thinking in terms of collections that will be created on the destination, then yes timeseries `CollectionMetadata` are now one-to-two. 

I think we should add a note to the comment explaining this.